static/js/table.js

```diff
@@ -74,7 +74,7 @@ class Table {
     } else {
       this.player().setName(getCookie('name'))
     }
-
+    document.getElementById('player-chip').innerText = this.player().setName(getCookie('name'))
```

Comment:
Ļoti viltīgi! Super, ar 1 rindu sanāca gan uzstādīit spēlētājam vārdu, gan spēlētāja čipā ierenderēt iekšā. 👍 !